tests/AspNetCore.Lambda.SampleApp/Person.fs

```diff
@@ -0,0 +1,7 @@
+namespace AspNetCore.Lambda.SampleApp
+
+//Model
+[<CLIMutable>]
+type Person = {
+    Name : string
+}
```

Comment:
This is only formatting, but to be consistent I'd prefer to stick with this style please:

```
type Person = 
    {
        Name : string
    }
```